mbuild/compound.py

```diff
@@ -1182,15 +1183,23 @@ def energy_minimization(self, steps=2500, algorithm='cg',
         """
         openbabel = import_('openbabel')
 
+        for particle in self.particles():
+            try:
+                elem.get_by_symbol(particle.name)
+            except KeyError:
+                raise MBuildError("Element name {} not recognized. Cannot "
+                                  "perform minimization."
+                                  "".format(particle.name)) from None
```

Comment:
Do you know if `from None` got backported to py27? A unit test that triggers the error regardless would be useful here.